stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StorageHelper.kt

```diff
@@ -47,7 +47,8 @@ public class StorageHelper {
     }
 
     public fun getFileAttachments(context: Context): List<AttachmentMetaData> {
-        return getFilteredAttachments(context, null)
+        // Excluding files with empty mime type just to be sure that we won't include folder and unknown files
+        return getFilteredAttachments(context, null).filterNot { it.mimeType.isNullOrEmpty() }
```

Comment:
Can't we exclude such file using projection (where in SQL)? Something like `MediaStore.FileColumns.MIME_TYPE != ""`

Reply:
Good idea, let me check that